src/Compilers/Core/Portable/MemberDescriptor.cs

```diff
@@ -1,10 +1,10 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
That is the unicode byte-order mark. It helps some editors figure out how to interpret the contents of the file. It doesn't need to be deleted.